### DISCUSSION TITLE
The initialization of an update causes a stutter during gameplay

### DISCUSSION THREAD

**Discussion by monochrome22** [ppy/osu#23624] [Ideas] [answered]
It happened to me twice that I was in the middle of gameplay and there was a stutter because an update was released and lazer started downloading it in the background. I didn't miss because of the stutter, but it feels like a frustrating moment waiting to happen. 
Either the update shouldn't initialize during gameplay or it should work in a way that doesn't cause stutters during gameplay.

**Comment by peppy** [comment, marked as answer]
https://github.com/ppy/osu/pull/23636
